src/python/WMComponent/TaskArchiver/TaskArchiverPoller.py

```diff
@@ -470,6 +463,47 @@ def notifyWorkQueue(self, subList):
 
         return
 
+    
+    def deleteWorkflowFromWMBSAndDisk(self):
```

Comment:
Don't you need to call this method in the algorithm()? Otherwise it won't run ever